### PR TITLE
fix: crash accounts if not selected output

### LINKED ISSUE #8205: [Task]: fix crash error when account management view is without account information
### Task description

fix crash error when account management view is without account information

### PR DESCRIPTION
**Pull request by cpl121**
Closes #8205 

## Summary

> Please summarize your changes, describing **what** they are and **why** they were made.

...

## Changelog

```
- Please write a list of changes
```

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/desktop/views/dashboard/account-management/AccountManagement.svelte [hunk 1]
```diff
@@ -28,19 +28,21 @@
...
+                            allOutputs={allAccountOutputs}
+                            {selectedOutput}
+                        />
+                    {/if}
+                    {#if features.accountManagement.accountDetails.enabled}
+                        <AccountManagementDetails {selectedOutput} index={setAccountOutputIndex(selectedOutput)} />
+                    {/if}
+                {/key}
+            </div>
+        {/if}
```
Comment (marc2332) [inline packages/desktop/views/dashboard/account-management/AccountManagement.svelte:46]: else add a message in an `else` so the view isn't empty

**Review by marc2332 — CHANGES_REQUESTED** [review]

File: packages/shared/locales/en.json [hunk 2]
```diff
@@ -715,7 +715,8 @@
                     "unsetAsPrimary": "Unset as primary"
                 }
             },
-            "notification": "Activate one account to access the dashboard."
+            "notification": "Activate one account to access the dashboard.",
+            "emptyAccounts": "There is no account to show, activate one first"
         },
         "delegation": {
             "title": "Delegation",
```
Comment (marc2332) [inline packages/shared/locales/en.json:722]:
```suggestion
            "emptyAccounts": "There is no account to show, activate one first."
        },
        "delegation": {
            "title": "Delegation",
```

**Review by marc2332 — CHANGES_REQUESTED** [review]

File: packages/shared/locales/en.json (hunk 2 above)
Comment (marc2332) [reply packages/shared/locales/en.json:722]: the rest LGTM

**Review by marc2332 — APPROVED** [review]

File: packages/shared/locales/en.json [hunk 3]
```diff
@@ -715,7 +715,8 @@
                     "unsetAsPrimary": "Unset as primary"
                 }
             },
-            "notification": "Activate one account to access the dashboard."
+            "notification": "Activate one account to access the dashboard.",
+            "emptyAccounts": "There is no account to show, activate one first."
```
Comment (evavirseda) [inline packages/shared/locales/en.json:719]:
```suggestion
            "emptyAccounts": "There are no accounts to show, activate one first."
```

**Review by evavirseda — CHANGES_REQUESTED** [review]

**Review by begonaalvarezd — APPROVED** [review]

**Pull request merged**